sample-apps/segmentation_spleen_scribbles/main.py

```diff
@@ -33,10 +33,10 @@ def init_infers(self):
             "Spleen_Segmentation": SegmentationWithWriteLogits(
                 self.final_model, load_from_mmar(self.mmar, self.model_dir)
             ),
-            "BIFSeg+CRF": SpleenBIFSegCRF(),
-            "BIFSeg+SimpleCRF": SpleenBIFSegSimpleCRF(),
-            "BIFSeg+GraphCut": SpleenBIFSegGraphCut(),
-            "Int.+BIFSeg+GraphCut": SpleenInteractiveGraphCut(),
+            "ISeg+CRF": SpleenISegCRF(),
+            "ISeg+SimpleCRF": SpleenISegSimpleCRF(),
+            "ISeg+GraphCut": SpleenISegGraphCut(),
+            "Int.+ISeg+GraphCut": SpleenInteractiveGraphCut(),
```

Comment:
`Int.+ISeg+GraphCut` => There is a dot..  
`Int.` doesn't sound normal..

Reply:
Fixed